### PR TITLE
Split-View implementation.

### PR DESCRIPTION
**Pull request by hemikak**

### CODE REVIEW

File: modules/web/js/ballerina/views/ballerina-file-editor.jsx [hunk 1]
```diff
@@ -78,7 +78,9 @@ class BallerinaFileEditor extends React.Component {
             if (originEvtType === CHANGE_EVT_TYPES.TREE_MODIFIED) {
                 // Change was done from design view
                 // do an immediate update to reflect tree changes
-                this.forceUpdate();
+                if (this.ballerinaFileEditorWrapper) {
```
Comment (kaviththiranga) [inline modules/web/js/ballerina/views/ballerina-file-editor.jsx]: Why do we need a ref check?

Comment (kaviththiranga) [reply modules/web/js/ballerina/views/ballerina-file-editor.jsx]: just wanted to know the issue we are trying to solve here

Comment (hemikak) [reply modules/web/js/ballerina/views/ballerina-file-editor.jsx]: They have to be removed. Will be removing them in the next PR.

Comment (hemikak) [reply modules/web/js/ballerina/views/ballerina-file-editor.jsx]: Fixed with https://github.com/ballerinalang/composer/pull/3086/commits/5c737891cc1aa309dd89713ba98a8568966ee51f

**Pull request merged**